orphedomos-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -7,23 +7,46 @@
 			<implementation>
 				org.apache.maven.lifecycle.mapping.DefaultLifecycleMapping
 			</implementation>
+		</component>
+		<component>
+			<role>org.apache.maven.lifecycle.Lifecycle</role>
+			<implementation>org.apache.maven.lifecycle.Lifecycle</implementation>
+			<role-hint>orphedomos</role-hint>
+			<configuration>
+				<id>default</id>
+				<phases>
+					<phase>validate</phase>
+					<phase>package</phase>
+					<phase>install</phase>
+					<phase>deploy</phase>
+				</phases>
+				<default-phases>
+					<validate>
+						org.technologybrewery.orphedomos:orphedomos-maven-plugin:verify-docker-environment
+					</validate>
+					<package>
+						org.technologybrewery.orphedomos:orphedomos-maven-plugin:build-docker-image
```

Comment:
I: Considering moving this to the install phase, and only retaining the function that sets the additional artifacts in the `package` phase.  It doesn't _really_ matter, but may be slightly more correct.

Reply:
Right.  The logic here is that we get pom publishing for free from the maven-install-plugin and maven-deploy-plugin, but if we don't specify some sort of primary artifact in addition, it'll throw an error.  I figure the dockerfile is as good an option as anything for that role, though I could also lean towards something like a flat file with a list of all produced tags.

Reply:
Per offline convo, pushed an update that aligns this w/ the pattern we used in habushu.  Pushes a placeholder file:
```
[INFO] --- install:3.1.1:install (default-install) @ orphedomos-test-multimodule-b ---
[INFO] Installing /Users/****/dev/technology-brewery/orphedomos/orphedomos-test/orphedomos-test-multimodule/orphedomos-test-multimodule-b/pom.xml to /Users/****/.m2/repository/org/technologybrewery/orphedomos/orphedomos-test-multimodule-b/0.8.0-SNAPSHOT/orphedomos-test-multimodule-b-0.8.0-SNAPSHOT.pom
[INFO] Installing /Users/****/dev/technology-brewery/orphedomos/orphedomos-test/orphedomos-test-multimodule/orphedomos-test-multimodule-b/target/orphedomos.placeholder.txt to /Users/****/.m2/repository/org/technologybrewery/orphedomos/orphedomos-test-multimodule-b/0.8.0-SNAPSHOT/orphedomos-test-multimodule-b-0.8.0-SNAPSHOT.orphedomos.placeholder.txt
```